streaming/src/test/java/org/apache/spark/streaming/JavaAPISuite.java

```diff
@@ -390,6 +390,7 @@ public void testQueueStream() {
     JavaTestUtils.attachTestOutputStream(stream);
     List<List<Integer>> result = JavaTestUtils.runStreams(ssc, 3, 3);
     Assert.assertEquals(expected, result);
+    jsc.close();
```

Comment:
These types of changes are welcome of course but I think we need to call `stop()` in the examples, not `close()`. Or at least that's what the others do. I don't think Coverity will realize that, but that's fine, it's then a false positive we just resolve as such.